apscheduler/triggers/base.py

```diff
@@ -8,6 +8,17 @@ class BaseTrigger(six.with_metaclass(ABCMeta)):
 
     __slots__ = ()
 
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
+        for slot in self.__slots__:
```

Comment:
In the case of built-in triggers, sure. But what if someone subclasses those triggers and adds their own `__slots__`?